tests/lib/devicetree/api/src/main.c

```diff
@@ -1270,6 +1270,18 @@ static void test_enums_required_false(void)
 #define DT_DRV_COMPAT vnd_adc_temp_sensor
 static void test_clocks(void)
 {
+	/* DT_CLOCKS_CTLR_BY_IDX */
+	zassert_true(!strcmp(TO_STRING(DT_CLOCKS_CTLR_BY_IDX(TEST_TEMP, 1)),
+			     TO_STRING(DT_NODELABEL(test_fixed_clk))), "");
```

Comment:
Any reason not to use DT_SAME_NODE? I think that reads a bit clearer,

Reply:
Just copy and pasted from another test, will look at using `DT_SAME_NODE`